ui/mirage/handlers/kubernetes.js

```diff
@@ -0,0 +1,68 @@
+import { Response } from 'miragejs';
+
+export default function (server) {
+  const getRecord = (schema, req, dbKey) => {
```

Comment:
Do these methods need to be defined within the server function? This might read a little clearer if they are defined outside of it

Reply:
`createRecord` uses the `server` instance but I suppose that could be passed in. Not sure if it's worth moving outside to then pass in the server instance though? I'm curious if there is a best practice regarding this or personal preference (thinking about possible eslint warning)?